test/unit/modules/blocks.js

```diff
@@ -6,7 +6,6 @@ var expect = require('chai').expect;
 var express = require('express');
 var sinon = require('sinon');
 
-var modulesLoader = require('../../common/modulesLoader');
```

Comment:
Is this is removed because of no usage? There are few other unused variables in this file.

Reply:
I'm removing just the one related to the scope of this PR.